4TT/sequential_doas_4TT.ipynb

```diff
@@ -330,7 +336,9 @@
     "for ax in axes.flatten():\n",
     "    ax.axhline(0, color=\"k\", linewidth=1)\n",
     "    ax.set_xlabel(\"Time (ps)\")\n",
-    "    ax.set_ylabel(\"\")"
+    "    ax.set_ylabel(\"\")\n",
+    "    ax_title = ax.get_title()\n",
+    "    ax.set_title(rf\"{ax_title.replace('spectral', 'Wavelength')}$\\,$nm\")"
```

Comment:
If replacing title spectral with wavelength, perhaps also change x-axis from time to Time (ps).  

I wouldn't mind the fontsize a little bigger.

Reply:
This is one example, but there are multiple cases in the multiple notebooks.

Reply:
We are not using any `plot_overview`plot in the publication, but we should add a more convenient way to change axis labels and titles on bulk in the extras at some point.